src/script/page/AppMain.tsx

```diff
@@ -35,6 +35,7 @@ import {PanelEntity, PanelState, RightSidebar} from './RightSidebar';
 import {RootProvider} from './RootProvider';
 import {useAppMainState, ViewType} from './state';
 
+import {UserModal} from '../components/Modals/UserModal';
```

Comment:
Let's move the import right next to the other components `import` 

```suggestion
import {UserModal} from 'Components/Modals/UserModal';
```